### ISSUE TITLE
no formal release for Rails 5 (yet)

### ISSUE THREAD

**Issue by ConfusedVorlon** [activescaffold/active_scaffold#516]
Hi,

I hope you won't mind me posting this as an issue - just as an easy way to keep track (and get the github notification) when you do post a formal release that supports rails 5.

My hope is that you'll close this issue when we can update our gemfiles to use a version number rather than git:...

I am aware that the master branch supports Rails 5, and I'm using it in production.

thank you for the ongoing support.

**Comment by ralfebert** [comment]
Using the current master branch in the Gemfile:

```
gem "active_scaffold", github: "activescaffold/active_scaffold", branch: "master"
```

doesn't work in a newly created Rails 5.0.0.1 app:

```
bin/rails g active_scaffold User name:string
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <class:Base> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/action_controller_rendering.rb:17)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <class:LookupContext> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/action_view_rendering.rb:8)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <module:RenderingHelper> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/action_view_rendering.rb:125)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from block in included at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/name_option_for_datetime.rb:10)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from block in <top (required)> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/paginator_extensions.rb:8)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from block in <top (required)> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/paginator_extensions.rb:26)
Running via Spring preloader in process 67594
Could not find generator 'active_scaffold'. Maybe you meant 'scss:scaffold', 'css:scaffold' or 'scaffold'
Run `rails generate --help` for more options.
```

any hints?

**Comment by CCH4GITHUB** [comment]
ConfusedVorlon: I am aware that the master branch supports Rails 5, and I'm using it in production.
Are you having issues with API:Column. I am not able to get AS to honour the API ?

**Comment by scambra** [comment]
@CCH4GITHUB your problem is not using argument in block, argument is conf and you use config:

wrong:
```ruby
active_scaffold do |conf|
  config.columns
```

Good:
```ruby
active_scaffold do |conf|
  *conf*.columns
```

**Comment by mcr** [comment]
With a Gemfile that was previously working great for rails 4.2, I get for both rails 5.0 and 5.1 with master branch:

    NameError: uninitialized constant ActiveRecord .../gems/ruby-2.4.1/bundler/gems/active_scaffold-9cbfb98bd39b/lib/active_scaffold/extensions/to_label.rb:2:in `<top (required)>'
 
I added:

   require 'active_record'

to the top of to_label.rb, and the problem went away.  I am guessing something to do with include path order maybe.

**Comment by scambra** [comment]
No issue here with rails 5.0 and master branch

**Comment by jbeker** [comment]
Is there a plan for a next official release? I don't like pining to `master` in my production environment if at all possible.

**Comment by brosert** [comment]
The 'to_label.rb' mod didn't seem to make a difference - and because I'm syncing with heroku not sure modifying the gem is the right approach....

is there any other gem we should be including on rails 5 (5.1.5), or is perhaps we've missed someit else?

getting:
Gem Load Error is: This version of ActiveScaffold requires Rails 3.1 or higher.  Please use an earlier version.
Backtrace for gem load error is:
lib/ruby/gems/2.4.0/gems/active_scaffold-3.2.20/lib/active_scaffold.rb:2:in `<top (required)>'

(on win10 if it makes a difference)

**Comment by scambra** [comment]
@brosert You are using a really old gem, gem supporting rails 5.0 or 5.1 is not relased, you must use master, although is really stable and will be released soon

**Comment by scambra** [comment]
released 3.5.0, it works with rails 5, and I think it may work with rails 5.1, although I haven't checked yet.
